content/influxdb/v2.0/upgrade/v1-to-v2/automatic-upgrade.md

```diff
@@ -101,10 +101,14 @@ You can continue to use Kapacitor with InfluxDB OSS 2.0 under the following scen
 ### User migration
 
 `influxd upgrade` migrates existing 1.x users and their permissions.
-However, it *does not migrate administrative users*.
+However, it **does not** migrate the following users:
```

Comment:
`influxd upgrade` migrates existing 1.x users and their permissions **except** the following users: